snuba/migrations/groups.py

```diff
@@ -82,12 +83,16 @@ def __init__(
         storage_sets_keys={StorageSetKey.MIGRATIONS},
         readiness_state=ReadinessState.COMPLETE,
     ),
+    MigrationGroup.CDC: _MigrationGroup(
+        loader=SystemLoader(),
```

Comment:
We might need to define a different loader here

Reply:
you're so right bestie